src/wwwroot/partials/settings/dkim-configuration-help.html

```diff
@@ -10,9 +10,7 @@
   <hr class="default" />
 </div>
 <div class="item section--wrapper flex">
-  <h2>
-    {{ 'dkim_configuration' | translate }}
-  </h2>
+  <h3><strong>{{ 'dkim_configuration' | translate }}</strong></h3>
```

Comment:
Is _strong_ necessary? I mean, are there other _h3_ headers without strong? if not, why not adding bold style to _h3_? if there are, why? @mvillaseco  what do you think?